src/client/settings/PersonalSettingsReducer.ts

```diff
@@ -89,14 +94,21 @@ const slice = createSlice({
 })
 
 export const { reducer } = slice
-export const { setAllowAudioNotifications, setAllowSystemNotifications } =
-  slice.actions
+export const {
+  setAllowAudioNotifications,
+  setAllowSystemNotifications,
+  setAudioNotificationVolume,
+} = slice.actions
 
 const getPersonalSettingsRoot = (state: RootState) => state.personal
 export const getAllowAudioNotifications = createSelector(
   getPersonalSettingsRoot,
   (settings) => settings.allowAudioNotifications
 )
+export const getAudioNotificationVolume = createSelector(
+  getPersonalSettingsRoot,
+  (settings) => settings.audioNotificationVolume || 1
```

Comment:
This is causing the max volume bug when you mute.
This is because 0 is "false" so 1 is being returned.
[A quick fix is changing `||` to `??` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)

Reply:
Brain dump thoughts you can probably ignore! 🤔 

Ideally you shouldn't have to return a default value from the getter, as in real life a value will always be returned.

The problem is the use of `t.partial` in the type definition. 

The reason most reducers use the `t.partial` is so that you can import incomplete/partial settings. The reason this is desirable is so that I didn't need to add a migration for every new reducer value.

Reply:
Aaaaah this makes a lot of sense, thanks for the context! It was tripping me up a bit 👍 

Would you like me to investigate a little as part of this PR, or are you happy-ish with the default value for now?

Reply:
Great spot, I 100% meant to null guard not falsey check, sorry!

Reply:
Happy to go with the default for this PR, to be consistent you'd have to add a "configuration" type to personal settings. This would result in a cascade of changes, and honestly just having a default value in getters might be a better idea 🤔 🤷